### DISCUSSION TITLE
Changing 'survey' to 'form'

### DISCUSSION THREAD

**Discussion by michielklonhammer** [zotonic/zotonic#3412] [Q&A]
At LearningStone we have worked a lot on the surveys and have named these "Forms" and we have overruled what is now Survey "Names" (the names of each survey item) to "Labels" as names was confusing.
I am currently working on the Dutch translations in Zotonic and suggest we change the word Survey to Forms as it makes more sense and Names to labels. 

Agree?

There are only 20 strings and most of them can be neutral like 
Thank you for filling in our survey. >>> Thank you for filling this in.
Because you can edit the survey, you can proceed for testing. >>> Because you can edit this, you can proceed for testing.

**Comment by DorienD** [comment]
I think "survey" to "form" would be an excellent solution especially since it can be used for more than only surveys now. What is a survey item? I'm wondering in which context "Names" is being used.

**Comment by michielklonhammer** [comment]
"Names" is this:
![image](https://github.com/zotonic/zotonic/assets/1306897/edaeb833-27e8-410d-ad82-4a1f65d8478b)
There are a few references to this, like "Questions with the name “email” can be exported."
We have overruled this with "Questions with the label “email” can be exported."

**Reply by DorienD** [reply]
Ah! totally agree with the proposed Names refactor.

**Comment by mworrell** [comment]
People find "label" confusing?  Interesting...

Could we use "question name" instead? As there are many other names, like the unique resource name and the person name.

I am ok with renaming the Surveys to Forms, as they are used for a lot more things than just surveys.

**Reply by michielklonhammer** [reply]
No the "name" is confusing (that's how it is now). I'm proposing to call it Question "label"

**Reply by michielklonhammer** [reply]
So Survey >>> Forms
Name >> Label  (Question label)

**Reply by mworrell** [reply]
Ah, the other way around.

Totally agree!

**Comment by geoidesic** [comment]
Is this completed? It's more than a year old. If so I suggest close the discussion.

**Comment by mworrell** [comment]
I think this is an actionable item for which we should make a clear issue.
